mmv1/third_party/terraform/services/artifactregistry/data_source_artifact_registry_docker_image_test.go

```diff
@@ -28,25 +28,12 @@ func TestAccDataSourceArtifactRegistryDockerImage(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName+"Tag", "image_size_bytes"),
 					validateTimeStamps(resourceName+"Tag"),
 
-					// Data source using a digest
-					checkDigestDataSources(
-						resourceName+"Digest",
-						"projects/cloudrun/locations/us/repositories/container/dockerImages/hello@sha256:7a6e0dfb0142464ce0ba14a2cfcac75e383e36f39f47539c870132c826314ad6",
-						"us-docker.pkg.dev/cloudrun/container/hello@sha256:7a6e0dfb0142464ce0ba14a2cfcac75e383e36f39f47539c870132c826314ad6",
-					),
 					resource.TestCheckResourceAttrSet(resourceName+"Digest", "image_size_bytes"),
 					validateTimeStamps(resourceName+"Digest"),
 
 					// url safe docker name using a tag
 					checkTaggedDataSources(resourceName+"UrlTag", "latest"),
-
-					// url safe docker name using a digest
-					checkDigestDataSources(
-						resourceName+"UrlDigest",
-						"projects/go-containerregistry/locations/us/repositories/gcr.io/dockerImages/krane%2Fdebug@sha256:26903bf659994649af0b8ccb2675b76318b2bc3b2c85feea9a1f9d5b98eff363",
-						"us-docker.pkg.dev/go-containerregistry/gcr.io/krane/debug@sha256:26903bf659994649af0b8ccb2675b76318b2bc3b2c85feea9a1f9d5b98eff363",
-					),
-
+					
```

Comment:
could you address the lint issues on your changes as well as provide a changelog that matches the correct [release notes docs format](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/)